web/src/lib/components/faces-page/set-birth-date-modal.svelte

```diff
@@ -31,6 +50,10 @@
       </p>
     </div>
 
+    {#if error}
+      <p class="text-sm text-red-500">{error}</p>
+    {/if}
+
     <form on:submit|preventDefault={() => handleSubmit()} autocomplete="off">
       <div class="m-4 flex flex-col gap-2">
         <input class="immich-form-input" id="birthDate" name="birthDate" type="date" bind:value={birthDate} />
```

Comment:
Can you add a `max` value in the date picker ?

Reply:
Thanks @martabal. The date picker will now have max value of today.